### DISCUSSION TITLE
Dark Mode for official docs

### DISCUSSION THREAD

**Discussion by damvaz** [quasarframework/quasar#14152] [Ideas / Proposals] [answered]
# Dark Mode for official docs

This feature is important because much of the documentation involves several minutes of screen reading.

**_Dark mode allows you to rest your eyes._**

On the other hand, many people have dark mode configured on their system, but currently the official documentation does not cater for that fact.

Still, it's important to include a switcher for people who want to set clear mode as an exception to their global preferences.

Dark mode involves changes not only to the color palette, but to the "dark versions" of the images in the documentation.

## Related PR
[14153](https://github.com/quasarframework/quasar/pull/14153)

**Comment by yusufkandemir** [comment, marked as answer]
This has been addressed in Quasar website GUI update: #13177, waiting to be reviewed and released later.
It's also mentioned in this old issue: https://github.com/quasarframework/quasar/issues/7881#issuecomment-1148173890

**Reply by damvaz** [reply]
My PR has no conflicts ;-)

And is fully compatible with current Quasar Docs.
